README.md

```diff
@@ -1,12 +1,14 @@
 Chameleon System SanityCheckBundle
 ==================================
 
-The SanityCheckBundle adds support for checks that aid you in finding common problems when running your application.
-For example, you can ensure that there is enough disk space, or permissions on certain files are granted. With this bundle
-you can configure checks that verify that the requirements for running the applications are met.
+The SanityCheckBundle is a Symfony integration bundle for the SanityCheck library.
 
-Where these checks are performed is up to you. In an installer, from the command line or an admin interface - just add
-the checks wherever they suit your needs.
+SanityCheck adds diagnosis capabilities to any PHP application. By defining a range of checks, it can be ensured that
```

Comment:
Maybe use the same wording as here https://github.com/chameleon-system/sanitycheck/pull/2

Reply:
Changed.